snippets/snip-private-clusters-public-ingress.adoc

```diff
@@ -0,0 +1,12 @@
+// Text snippet included in the following modules:
+//
+// * modules/private-clusters-default.adoc
+// * modules/private-clusters-about.adoc
+// * modules/private-clusters-about-aws.adoc
+
+:_content-type: SNIPPET
+
+[IMPORTANT]
+====
+If the cluster has any public subnets, load balancer services created by administrating users might be publicly accessible. To ensure cluster security, verify that these services are explicitly annotated as private.
```

Comment:
Does `administrating users` mean `administrators` here?

Reply:
Yes thank you, admin user--administrator same thing. I think I can just say administrator in the doc.